src/getImports/babelParser.js

```diff
@@ -26,13 +30,39 @@ export function getPackages(fileName, source, language) {
   const packages = [];
   const visitor = {
     ImportDeclaration({ node }) {
-      packages.push({
-        fileName,
-        loc: node.source.loc,
-        name: node.source.value,
-        line: node.loc.end.line,
-        string: compileImportString(node),
-      });
+      const configuration = workspace.getConfiguration('vulnCost');
+      let pathIgnored = false;
+
+      for (let i = 0; i < configuration.ignorePaths.length; i++) {
+        const path = configuration.ignorePaths[i];
+        pathIgnored = new RegExp(path).test(node.source.value);
+
+        if (pathIgnored) {
+          continue;
```

Comment:
Isn’t the intention to break out of the `for` loop here?
```suggestion
          break;
```